org.virt_manager.virt-viewer.yml

```diff
@@ -0,0 +1,186 @@
+app-id: org.virt_manager.virt-viewer
+runtime: org.gnome.Platform
+runtime-version: "43"
+sdk: org.gnome.Sdk
+command: remote-viewer
```

Comment:
The original proposal in #391 had this as 'virt-viewer', with the intent that running from the cli eg 'flatpak run org.virt_manager.virt-viewer <args>' would spawn virt-viewer, while for remote-viewer you could just select the desktop icon from your desktop menu. I'm not sure if that actually worked in practice, but it feels a little more desirable.

If we're calling the app 'virt-viewer' then we ought to be able to run the virt-viewer binary with it.

Reply:
the command has to match the executable in the .desktop file. However you can export more than one which could fit that need.